.github/workflows/docker_publish.yml

```
@@ -8,15 +8,15 @@ on:
       - 'docker/*'
       - '.github/workflows/docker_publish.yml'
 
-env: 
-  DOCKER_IMAGE_BASENAME: ghcr.io/${{ github.repository_owner }}/ubuntu_20.04_py3
+env:
+  DOCKER_IMAGE_BASENAME: ghcr.io/${{ github.repository_owner }}/pyne/ubuntu_20.04_py3
```

I am not confident in how all these directory-like names work.  I think it would be better if the basename was
```suggestion
  DOCKER_IMAGE_BASENAME: ghcr.io/${{ github.repository_owner }}/pyne_ubuntu_20.04_py3
```